frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetInstallSubmariner/InstallSubmarinerForm.tsx

```diff
@@ -362,8 +377,31 @@ export function InstallSubmarinerForm(props: { availableClusters: Cluster[] }) {
             })
         }
 
-        // if globalnet support create
-        if (!isGlobalnetAlreadyConfigured || !selectedClusters || !selectedClusters.length) {
+        if (anyUnsupported) {
+            resources.push({
+                apiVersion: ManagedClusterAddOnApiVersion,
+                kind: ManagedClusterAddOnKind,
+                metadata: {
+                    name: 'submariner',
+                    namespace: 'roks-cluster',
+                },
+                spec: {
+                    installNamespace: 'submariner-operator',
+                },
+            })
```

Comment:
@jeswanke Why is this required?
The ManagedClusterAddOnKind objects have already been created on line 332 for all unsupported clusters.
I don't think we should have this additional entry created in roks-cluster namespace.
In https://github.com/stolostron/backlog/issues/22622, roks-cluster is just the name (example) of one of the cluster  we are trying to install the submariner addons on.